python/cudf/cudf/core/column/string.py

```diff
@@ -3744,6 +3828,18 @@ def nvstrings(self):
     # TODO: Remove these once NVStrings is fully deprecated / removed
     @property
     def nvcategory(self):
+        warnings.warn(
+            "nvcategory will be removed in 0.15. \
+            Please use equivalent from libcudf",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+        warnings.warn(
+            "nvstrings will be removed in 0.15. \
+            Please use equivalent from libcudf",
+            DeprecationWarning,
+            stacklevel=2,
+        )
```

Comment:
```suggestion
        warnings.warn(
            "nvcategory will be removed in 0.15. \
            Please use equivalent from libcudf",
            DeprecationWarning,
            stacklevel=2,
        )
```

Reply:
One warning was for nvcategory and other for nvstrings as both are being used.

Reply:
I did _not_ notice that. Thank you. :)